src/main/java/uk/gov/justice/laa/crime/cfecrime/utils/RequestHandlerUtil.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.justice.laa.crime.cfecrime.utils;
+
+import lombok.experimental.UtilityClass;
+import lombok.extern.slf4j.Slf4j;
+import uk.gov.justice.laa.crime.cfecrime.api.Result.Outcome;
+import uk.gov.justice.laa.crime.cfecrime.api.*;
+
+@Slf4j
+@UtilityClass
+public class RequestHandlerUtil {
+
+    public static CfeCrimeResponse handleRequest(CfeCrimeRequest request){
+        CfeCrimeResponse response = new CfeCrimeResponse();
+        Boolean under18 = false;
+        Boolean passported = false;
+        Outcome oc = Outcome.INELIGIBLE;
+
+        if (request.getSectionUnder18() != null) {
+            under18 = request.getSectionUnder18().getClientUnder18();
+        }
+        if (request.getSectionPassportedBenefit() != null) {
+            passported = request.getSectionPassportedBenefit().getPassportedBenefit();
+        }
+        oc = getOutcomeFromAgeAndPassportedBenefit(under18,passported);
+
+        if (request.getAssessment() != null) {
+            Assessment assessment = request.getAssessment();
+            if (assessment.getAssessmentDate() != null) {
+                oc = Outcome.ELIGIBLE;
```

Comment:
IMHO There's nothing in the requirements that makes the outcome eligible for this scenario